appengine-standard-java8/springboot-appengine-standard/src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Google Inc.
```

Comment:
This is supposed to be "Google LLC" now (this just started recently).

Reply:
Done.